Sources/Datadog/RUM/AutoInstrumentation/LongTasks/LongTaskObserver.swift

```diff
@@ -0,0 +1,83 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import Foundation
+
+internal class LongTaskObserver {
+    let longTaskDurationThreshold: TimeInterval
+    let dateProvider: DateProvider
```

Comment:
Looks like it can be `private`:
```suggestion
    private let longTaskDurationThreshold: TimeInterval
    private let dateProvider: DateProvider
```